### PR TITLE
COMMERCE-1748 add to cart in product details behaves like in catalog

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**

### CODE REVIEW

File: commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/product/renderer/list/entry/MiniumCPContentListEntryRenderer.java [hunk 1]
```diff
@@ -273,11 +273,11 @@ public void render(
...
 
 		dependencies.add(
-			"commerce-frontend-taglib@2.1.6/add_to_cart/AddToCartButton.es");
+			"commerce-frontend-taglib@2.1.7/add_to_cart/AddToCartButton.es");
 		dependencies.add(
-			"commerce-frontend-taglib@2.1.6/compare_checkbox" +
+			"commerce-frontend-taglib@2.1.7/compare_checkbox" +
 				"/CompareCheckbox.es");
-		dependencies.add("commerce-frontend-taglib@2.1.6/price/Price.es");
+		dependencies.add("commerce-frontend-taglib@2.1.7/price/Price.es");
```
Comment (gianmarcobrunialti) [inline commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/product/renderer/list/entry/MiniumCPContentListEntryRenderer.java:281]: These changes reflect the frontend taglib version update (https://github.com/liferay/com-liferay-commerce/blob/7.1.x-next/commerce-frontend-taglib/bnd.bnd#L3)

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by gianmarcobrunialti** [comment]
ci:test

File: commerce-theme-minium/commerce-theme-minium/static-version/src/styles/components/add-to-cart-button.scss [hunk 2]
```diff
@@ -198,21 +198,45 @@
...
+				display: none;
+			}
 
+			&.commerce-button--good {
 				&:hover::before {
 					align-items: center;
+					animation: addMoreHover .1s ease-out 1 forwards;
+					background-color: $color-theme;
+					border-radius: 8px;
 					content: attr(data-add-more-label);
```
Comment (marco-leo) [inline commerce-theme-minium/commerce-theme-minium/static-version/src/styles/components/add-to-cart-button.scss:213]: @gianmarcobrunialti what is doing this line : content: attr(data-add-more-label); ?

Comment (gianmarcobrunialti) [reply commerce-theme-minium/commerce-theme-minium/static-version/src/styles/components/add-to-cart-button.scss:213]: It's grabbing the data-attribute (`data-add-more-label`) from the element (rendered through the soy template, by which we are using i18n properties) to populate the content of its `::before` pseudo-element.

File: commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AddToCartTag.java [hunk 3]
```diff
@@ -69,12 +71,20 @@ public int doStartTag() {
 
 			CommerceOrder commerceOrder = commerceContext.getCommerceOrder();
 
+			int productOrderQuantity = 0;
+
 			if (commerceOrder != null) {
 				putValue("orderId", commerceOrder.getCommerceOrderId());
+
+				List<CommerceOrderItem> commerceOrderItems =
```
Comment (marco-leo) [inline commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AddToCartTag.java:79]: @alecsloan  I am mergin this, but put in the backlog a change to use count from service

Comment (alecsloan) [reply commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/AddToCartTag.java:79]: @marco-leo @gianmarcobrunialti See my diff on pull #1348 for implementing new methods to get OrderItems by CPInstanceId

**Pull request merged**